packs/factions.db

```diff
@@ -40,8 +40,4 @@
-{"_id":"2K5r1qw5ksVTCTPG","name":"Citizenry","permission":{"default":0,"5izWgADOTEOz6xAL":3},"type":"faction","data":{"description":"The ordinary citizens of a district can be represented by a faction, if you want to track status with them in the game. The GM will set their Tier according to the wealth and power of the citizens in that district (Whitecrown might be Tier V, The Docks Tier II, Charhollow Tier 0).","type":"Citizenry","tier":"1","goal_1":"","goal_1_clock_max":0,"goal_2":"","goal_2_clock_max":0,"turf":"","assets":"","quirks":"","notables":"","allies":"","enemies":"","situation":"","goal_clock":0,"hold":{"value":1,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icon.1_39.png","notes":"","effects":[]}
\ No newline at end of file
+{"_id":"zs5YXCKaGGg8WhLU","name":"Leviathan Hunters","permission":{"default":0,"5izWgADOTEOz6xAL":3},"type":"faction","data":{"description":"The captains and crews that grapple with titanic demons of the Void Sea to drain their blood for processing into electroplasm.","type":"Institutions","tier":"5","goal_1":"Discover new hunting grounds","goal_1_clock_max":12,"goal_2":"Surplus runs dry","goal_2_clock_max":12,"turf":"The massive metal docks for the huge hunter ships and the associated construction and repair facilities. Several small private leviathan blood processing facilities for the captains’ personal shares.","assets":"The leviathan hunter fleet (each vessel is owned by the noble house who built and commands it). Many cohorts of expert sailors, as well as spark-craft technicians, demonologist Whispers, and void-touched navigators. Companies of marines to protect the vessels and their valuable cargo at sea and in port.","quirks":"","notables":"Lord Strangford (captain, ruthless, arrogant, tainted). Lady Clave (captain, daring, cruel, accomplished). Lady Ankhayat (Iruvian captain, confident, charming, scoundrel).","allies":"City Council, The Church of Ecstasy, Sailors, Dockers, Sparkwrights.","enemies":"The Grinders, Ministry of Preservation, The Path of Echoes.","situation":"The captains have a horrible secret: the known hunting grounds for leviathans are coming up barren. The immortal creatures, once so reliable in their movements in the Void Sea, have begun to migrate elsewhere. New hunting grounds must be found before the surplus of leviathan blood disappears, and with it, the lightning barriers and the survival of the human race.","goal_clock":0,"hold":{"value":2,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icons8_16.png","notes":"","effects":[]}
```

Comment:
Please make sure to have a newline in the end of the file  :)

Reply:
Added.